MTM-61325 changelog, operationsCreated and operationsUpdated counters… (#2601)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-10-20-611-0-usage-statistics-new-fields-counting-operations.md

---
date:
title: Added operation request counters to usage statistics
product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-OG_650_b2

>     label: Core platform

Author: I have found also component:
```
  - value: component-JlFdtOPva
    label: REST API
```
used in other change-log entries, but it is less used in 10.20 release. We can decide as a team which one fits better here.

Reviewer: I would use REST API here to be more precise.

Author: done, squashed in first commit

build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-61325
version: 10.20.611.0
---

> Two new properties: `operationsCreatedCount`, `operationsUpdateCount` were added to all REST endpoints returning usage statistics detailed requests counters. 

Reviewer: ```suggestion
Two new properties `operationsCreatedCount`, `operationsUpdateCount` have been added to all REST endpoints which will return detailed request counters for usage statistics. 
```

> Two new properties `operationsCreatedCount`, `operationsUpdateCount` have been added to all REST endpoints which will return detailed request counters for usage statistics. 

Reviewer: ```suggestion
Two new properties, `operationsCreatedCount` and `operationsUpdateCount`, have been added to all REST endpoints which will return detailed request counters for usage statistics. 
```

> At the first step of this new feature {{< product-c8y-iot >}} returns those values always equal `0`. 

Reviewer: ```suggestion
In a first implementation step, the {{< product-c8y-iot >}} platform returns these properties with `0` values. 
```

> Implementation of counting logic in reaction to REST requests and MQTT messages creating and updating operations will be added in the following changes to the 10.20 release.  

Reviewer: ```suggestion
In a future implementation, counting logic in reaction to REST requests and MQTT messages creating and updating operations will be added.  
```
